PhysicsTools/PatAlgos/plugins/PATPackedCandidateProducer.cc

```diff
@@ -135,6 +143,9 @@ pat::PATPackedCandidateProducer::PATPackedCandidateProducer(const edm::Parameter
   produces< std::vector<pat::PackedCandidate> > ();
   produces< edm::Association<pat::PackedCandidateCollection> > ();
   produces< edm::Association<reco::PFCandidateCollection> > ();
+
+  produces< edm::ValueMap<pat::HcalDepthEnergyFractions> > ("hcalDepthEnergyFractions");
```

Comment:
```if (not pfCandidateTypesForHcalDepth_.empty() )```
to match L441.
Otherwise we are in a somewhat bad state of declaring that a branch is produced; but then no data is added to the event.